### ISSUE TITLE
stop() not working and it is speaking even if the app is in background or killed

### ISSUE THREAD

**Issue by Nikhil-Daffodil** [vilic/cordova-plugin-tts#97]
In ionic 4 tts stop() is not working (but speak('') is working ) it is showing plugin_not_installed and also it is speaking even if the app is in background mode Or killed

**Comment by jbagaresgaray** [comment]
any update on this issues?

**Comment by Nikhil-Daffodil** [comment]
No, still not got any update and a new issue occurred. When you start tts and minimize the app OR even kill the app from the background, it will still be speaking until it finish the complete string

**Comment by hthieu1110** [comment]
I'd love to have the fix as well. The voice continues even when app get killed. Unfortunately I have not enough Android skill to submit a PR :(
